### ISSUE TITLE
[WARN] UDM cannot find UDR.

### ISSUE THREAD

**Issue by endrigoshi** [free5gc/free5gc#59]
While running TestRegistration, I noticed the following Warning in UDM:
```
0-06-02T16:53:37Z]/src/udm/udm_producer/ue_context_management.go:31 free5gc/src/udm/udm_producer.createUDMClientToUDR() Use default UDR Uri bacause ID[suci-0-208-93-0-0-0-00007487] does not match any UDR  UDM=Handler
```
In `udm/udm_producer/ue_context_management` I saw that in the `getUdrUri` method, there is no **if-condition** to check if the string contains `suci`, which then yields the above Warning.

I'm not sure if this is a bug or not, but I wanted to report it nonetheless.

**Comment by free5gc-org** [comment]
This is a bug, we'll fix it in our next release.

Thanks for your report!

**Comment by free5gc-org** [comment]
Dear @endrigoshi ,

free5GC has fix this bug on v3.0.3. Please try and check it.

Thanks for your report!
